README.md

```diff
@@ -1,112 +1,89 @@
 
-What if I find a bug?
-=====================
+## Help
 
-Please submit it to [Github](https://github.com/9wm/9wm/issues).
+There's a man page where you can consult information:
 
-If you can fix it, submit the fix as a pull request.
+```
+$ man 9wm
+```
 
+## Bugs
 
-Known Problems/Bugs
-===================
+Find a bug? Please, submit it to [issues]. If you can fix it, submit the fix as a pull request.
 
-9wm uses ideas from Plan 9's `rio`,
-but is following a different path and is not exactly the same.
+### Known Problems/Bugs
 
-In an attempt to avoid ludicrously long window names in the menu,
-9wm shortens names with colons and dashes in them.
-The algorithm works well for most programs I've tried,
-but is bound to fail with something or other,
-rendering a near-useless icon name.
+9wm uses ideas from Plan 9's rio, but is following a different path and is not exactly the same.
 
-Windows that provide their own placement information aren't handled well right now.
-I'm working on it.
+In an attempt to avoid ludicrously long window names in the menu, 9wm shortens names with colons and dashes in them. The algorithm works well for most programs I've tried, but is bound to fail with something or other, rendering a near-useless icon name.
 
-9wm has no idea how many monitors you have,
-nor what their dimensions are.
-It also can't deal with desktop geometry changes,
-like plugging in an external monitor.
-I run "9wm restart" to make it quickly reset its internal state and notice any changes.
+Windows that provide their own placement information aren't handled well right now. I'm working on it.
 
-9wm doesn't have Unicode support.
-In particular, if you have a browser page open with non-Latin characters in the name,
-and you hide that window,
-you will get a weird name for that window's entry in the list.
-Doing Unicode in X11 is a horrible mess requiring,
-among other things, a whole new library for rendering text.
-I may get to this in the future, but for now,
-I feel like learning about Wayland is a better use of my effort.
+9wm has no idea how many monitors you have, nor what their dimensions are. It also can't deal with desktop geometry changes, like plugging in an external monitor. I run "9wm restart" to make it quickly reset its internal state and notice any changes.
 
-Lack of EWMH means some applications (especially games) will break when 
-trying to go fullscreen.
+9wm doesn't have Unicode support. In particular, if you have a browser page open with non-Latin characters in the name, and you hide that window, you will get a weird name for that window's entry in the list. Doing Unicode in X11 is a horrible mess requiring, among other things, a whole new library for rendering text. I may get to this in the future, but for now, I feel like learning about Wayland is a better use of my effort.
```

Comment:
It doesn't look like any of the above have actually had any content changes, just spacing? If that's correct, please revert, it adds unnecessary noise to the diff

Reply:
That's right. I reverted the changes :)